cmake/HPX_PerformCxxFeatureTests.cmake

```diff
@@ -10,187 +10,6 @@
 # C++ feature tests
 ################################################################################
 function(hpx_perform_cxx_feature_tests)
-
-  # Check the availability of certain C++11 language features
-  hpx_check_for_cxx11_alias_templates(
-    REQUIRED "HPX needs support for C++11 alias templates")
-
-  hpx_check_for_cxx11_alignas(
-    DEFINITIONS HPX_HAVE_CXX11_ALIGNAS)
-
-  hpx_check_for_cxx11_auto(
-    REQUIRED "HPX needs support for C++11 auto")
-
-  hpx_check_for_cxx11_constexpr(
-    DEFINITIONS HPX_HAVE_CXX11_CONSTEXPR)
```

Comment:
What about `constexpr` support in NVCC? Does it support this sufficiently by now?

Reply:
CUDA + gcc on linux is fine. CUDA clang and MSVC are another question, but constexpr is ifdefed out for those as before. I wonder if those builds work anyway...